### PR TITLE
128 update domain page after saving updated records

### PR DESCRIPTION
**Pull request by eduramme**

### CODE REVIEW

File: pages/domains/[name]/index.tsx [hunk 1]
```diff
@@ -59,6 +63,12 @@ export function ManageNamePageContent({ name }: { name: string }) {
     handleFetchENSDomainData();
   }, []);
 
+  useEffect(() => {
```
Comment (lgahdl) [inline pages/domains/[name]/index.tsx:66]: You can move the "handleFetchENSDomainData()", to the "EditModalContent->closeModal" and avoid the usage of the hook useEffect

**Review by lgahdl — CHANGES_REQUESTED** [review]

File: pages/domains/[name]/index.tsx (hunk 1 above)
Comment (eduramme) [reply pages/domains/[name]/index.tsx:66]: @lgahdlIf I move handleFetchENSDomainData() to be called within the closeModal function, the page content will re-render every time the modal closes, even if there were no changes made to the data

Comment (lgahdl) [reply pages/domains/[name]/index.tsx:66]: Oh, I see, the updateFieldsWithEnsData function changes the layout of the fields, you're right. Ok then

**Review by lgahdl — APPROVED** [review]

File: pages/domains/[name]/index.tsx (hunk 1 above)
Comment (eduramme) [reply pages/domains/[name]/index.tsx]: @lgahdl Actually, you are right. useEffect will also rerender the data every time the modal closes. I updated the component so new info is only fetched  when necessary

**Review by FrancoAguzzi — APPROVED** [review]

**Pull request merged**